tests/AstRunner/AstMap/FileNameTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Qossmic\Deptrac\AstRunner\AstMap;
+
+use PHPUnit\Framework\TestCase;
+use Qossmic\Deptrac\AstRunner\AstMap;
+
+final class FileNameTest extends TestCase
+{
+    public function testPathNormalization(): void
+    {
+        $fileName = new AstMap\FileName('/path/to/file.php');
+        $this->assertSame('/path/to/file.php', $fileName->getFilepath());
```

Comment:
why? its not necessary for the current test.

it would only be necessary, in case you want to test the already existing functionality which was not changed by this PR

Reply:
You are right, it would help with test coverage. That is what I meant by "nice". Also, it makes the test fragile. (not really on GitHub but locally for sure). If I were to run the test in the `path` directory, it would fail mysteriously. It is a tiny thing, but it is IMHO good practice to mock out system dependencies, like the current working directory.